### DISCUSSION TITLE
BMCL论坛新注册的用户无法发帖

### DISCUSSION THREAD

**Discussion by Blue-Beaker** [bangbang93/openbmclapi#35] [Q&A]
在BMCL论坛新注册了用户并通过收到的验证邮件验证邮箱，然后登录，找不到发帖按钮，无法发帖。一个多月后再进论坛，仍没有发帖按钮。尝试邮件联系后没有回应，只能在这里求助。

**Comment by bangbang93** [comment]
你需要完善并且验证一下邮箱

**Reply by Blue-Beaker** [reply]
注册完几分钟内就验证了邮箱，再次验证则返回"链接已失效"，能申请重新验证吗

**Reply by bangbang93** [reply]
应该可以自己重发验证码的

**Reply by Blue-Beaker** [reply]
没找到重发验证的入口，通过更换邮箱的方式重新验证通过后也无效

**Reply by bangbang93** [reply]
我发现我的邮件配置有问题……我现在比较好奇你是怎么收到验证码的
